### PR TITLE
deps: bump celestia-app to v0.13.0

### PR DESCRIPTION
**Pull request by jcstein**
This bumps `celestia-app` to v0.13.0. I will cut a new release when this is merged.

### CODE REVIEW

**Review by gupadhyaya — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/celestiaorg/celestia-app:v0.13.0 AS celestia-app
+FROM ghcr.io/celestiaorg/celestia-app:sha-6ba60e1 AS celestia-app
```
[the remark this answers is not in the document]
Comment (jcstein) [reply Dockerfile:1]: awesome, thank you for the context @rootulp!

**Comment by jcstein** [comment]
hm, @jrmanes do you know how we can work around [this](https://github.com/rollkit/local-celestia-devnet/actions/runs/4874782684/jobs/8696161092?pr=28) and [this](https://github.com/rollkit/local-celestia-devnet/actions/runs/4874782432/jobs/8696158711?pr=28) error? I think the docker publish workflow not being backported to 0.13.x may be the culprit here

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (nashqueue) [reply Dockerfile:1]: cc: @jrmanes

[the remark this answers is not in the document]
Comment (jcstein) [reply Dockerfile:1]: all good! if no good workaround, will use tags 😄

**Comment by jrmanes** [comment]
> 

Hey @jcstein 
I've checked the issue and as you said, the problem is that the image doesn't exist for `arm64` (and the pipeline is trying to pull that image), because the version `v0.13.0` doesn't contain the fix yet... 
There's nothing we can do until the fix will be added... 😓

**Comment by smuu** [comment]
Node `v0.9.2` is still using app `v0.12.0`. It would be compatible with `v0.13.0` but to avoid confusion let's wait for a release from node that uses app `v0.13.x`.

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (smuu) [reply Dockerfile:1]: I would prefer manually building and pushing `v0.13.0` and using the tag instead of using an SHA-tagged container.

**Comment by jcstein** [comment]
Sounds good, thanks team!

**Pull request closed without merging**